### ISSUE TITLE
Listen to all events globally, regardless of source?

### ISSUE THREAD

**Issue by ryanking1809** [alloc/ople#9]
Is there a way to listen to all events, regardless of the source? I suppose you would also want to emit the source with the event in that case.

**Comment by aleclarson** [comment]
Not yet. 

In the next version, I will re-export the `globals` object from `wana` and add `onEmit` to it. 👍 

Then you can do this:

```js
import {globals} from 'ople'

globals.onEmit = (source, event, args) => {}
```

**Comment by ryanking1809** [comment]
Perfect, thanks!

**Comment by aleclarson** [comment]
🎉 Released in `v0.4.3`

**Comment by aleclarson** [comment]
🎉 Released in `v0.4.3`
